packages/ez-react/src/recipes/bar/BarChart.tsx

```diff
@@ -93,6 +99,7 @@ export const BarChart: FC<BarChartProps> = ({
             domainKey: xAxis.domainKey,
             nice: xAxis.nice || 0,
             reverse: isRTL,
+            domain: [0, xAxisDomainMaxValue],
```

Comment:
```suggestion
            nice: typeof xAxis.nice !== 'undefined' ? xAxis.nice : 1,
            reverse: isRTL,
```
Note that values could be negative, so we cannot set zero as lower band. By default the scale will compute the min and max by using the xAxis.domainKey, so there's no need to re-calculate the max value. I think we should set nice to "1" if none is provided to solve the issue.